### PR TITLE
Feature/NEX-1529/remove test preview dependency

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/NEX-1529
Related to:
https://github.com/oat-sa/extension-tao-testqti/pull/1904
https://github.com/oat-sa/extension-tao-test/pull/348

**Issue:**
In working branches 
"oat-sa/extension-tao-test": "dev-feature/NEX-1043/test-preview-base-branch",  
"oat-sa/extension-tao-testqti": "dev-feature/NEX-1043/test-preview-base-branch", 
an issue with building bundles because of new dependency taoQtiTestPreview

**Solution:**
Dynamically load taoQtiTestPreviewer adapter

**How to test:**

1. checkout brances "oat-sa/extension-tao-test": "dev-feature/NEX-1043/test-preview-base-branch",  "oat-sa/extension-tao-testqti": "dev-feature/NEX-1043/test-preview-base-branch", 
2. in taoQtiTestPreviwer checkout to develop
3. open Tests page
4. press Preview button
5. in taoQtiTestPreviwer checkout to feature/NEX-1043/test-preview-base-branch
6. reload Tests page
7. press Preview button

### CODE REVIEW

**Review by jsconan — COMMENTED** [review]
The overall idea is ok. But it would have been better to rely on the taoTest extension to supply the API to manage the provider registration and management instead of directly write into the registry (cf. other PR).

**Comment by bziondik** [comment]
@jsconan The configuration will be handled in a separate ticket

**Comment by habemuscode** [comment]
Manifest version bump?

**Review by habemuscode — APPROVED** [review]
GJ!
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

File: views/build/grunt/bundle.js [hunk 1]
```diff
@@ -36,7 +36,9 @@ module.exports = function(grunt) {
                         name : 'qtiPreviewer',
                         babel : true,
                         include : [
-                            'taoQtiTestPreviewer/previewer/**/*'
+                            'taoQtiTestPreviewer/previewer/**/*',
+                            'taoQtiTestPreviewer/previewer/adapter/test/qtiTest',
+                            'taoQtiTestPreviewer/previewer/component/test/qtiTest'
```
Comment (jsconan) [inline views/build/grunt/bundle.js:41]: Is it needed? the line above contains wildcards, so it should not be needed to explicitly mention the sub-folders, unless there is a special trick due to the presence of the `/test/` folder?

**Review by jsconan — COMMENTED** [review]
The previewer is not set upon install, only on update. You should also add the installer in the manifest file, it is not called implicitly.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: views/build/grunt/bundle.js (hunk 1 above)
Comment (bziondik) [reply views/build/grunt/bundle.js:41]: yes, bundler doesn't include these 2 files because `/test/ `in the path

Comment (jsconan) [reply views/build/grunt/bundle.js:41]: ok, fine!

**Review by jsconan — APPROVED** [review]
GJ!

**Pull request merged**